### ISSUE TITLE
DKPY2: Arming motors message even when not arming

### ISSUE THREAD

**Issue by hamishwillee** [dronekit/dronekit-python#323]
I've updated simple_goto.py (see branch tmp_getting_started_dkpy2 which will be merged in #320).

If I re-run the script when the vehicle is airborne the system still reports ">>> ☺ARMING MOTORS" etc. Yes, the script does set ``vehicle.armed = True`` but it is already armed and flying and the vehicle will ignore this command. So ideally in this case the message should not be shown.

```
Arming motors
Taking off!
 Altitude:  20.0100002289
Reached target altitude
Going to first point...
>>> ☺ARMING MOTORS
>>> ☺Initialising APM...
```

Note, I appreciate that perhaps this should just be ignored. I'm just raising the issue so we make a decision and I can document the behaviour. Ditto with ``Initialising APM``. Other alternatives 

- change the message so that it is clear that we're sending a command only
- remove all auto-generated messages - let the script author define their own messages if needed.

I lean towards the last point, though if we were actually triggering off an arming message back from the autopilot , I might reconsider

**Comment by tcr3dr** [comment]
Since we can't change the message coming from APM, I would propose that we instead check if we are armed already. If we are, we do not set the arming command, thus preventing an excessive message from being sent.

**Comment by hamishwillee** [comment]
@tcr3dr Good idea.

**Comment by tcr3dr** [comment]
Fixed in #444.
